eldritch/encounters.py

```diff
@@ -22,7 +22,7 @@ def add_encounter(self, location_name, encounter_creator):
   def encounter_event(self, character, location_name):
     if location_name not in self.encounters:
       print(f"TODO: missing encounters for {self.name} at {location_name}")
-      return events.Nothing()
+      return events.Unimplemented()
```

Comment:
Only if none of the cards for a given location had the encounter implemented. Otherwise, `Encounter.resolve()` will just keep drawing cards until it finds one that works.